docs/CONFIGURATION.md

```diff
@@ -0,0 +1,188 @@
+# Configuring the Mountpoint for Amazon S3 CSI Driver
+
+See [the Mountpoint documentation](https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md) for
+Mountpoint specific configuration.
+
+## AWS Credentials
+
+The Mountpoint CSI Driver can be configured to ingest credentials via two approaches: globally for the entire 
+Kubernetes cluster, or using credentials assigned to pods.
+
+### Driver Level Credentials
+
+By setting driver-level credentials, the whole cluster uses the same set of credentials. 
+
+Using this configuration, the credentials that are used are set at installation time, either using Service
+Accounts, or using K8s secrets. 
```

Comment:
It would be nice to mention the order CRT uses to detect credentials. So customers would know e.g., if they set long-term credentials and IRSA, long-term credentials would always take precedence and IRSA won't be used.

Reply:
Agree, so long as they are configurations we're committed to support.

Probably I'd focus on the two mechanisms we intend to support, which is IRSA and Kubernetes secrets (and we recommend against K8s secrets).